src/utils.ts

```diff
@@ -44,11 +48,10 @@ export namespace Utils {
     timeout: number = 300,
     networkPassphrase: string,
     webAuthDomain: string,
+    memo: string | null = null,
   ): string {
-    if (clientAccountID.startsWith("M")) {
-      throw Error(
-        "Invalid clientAccountID: multiplexed accounts are not supported.",
-      );
+    if (clientAccountID.startsWith("M") && memo) {
```

Comment:
I recommend using `isValidMed25519PublicKey(clientAccountID)` which should be a little more robust (albeit less performant).

Reply:
The base repository doesn't expose that function on `StrKey`: https://github.com/stellar/js-stellar-base/blob/66745adf447a9544aeca612559b38953fd0284a3/types/index.d.ts#L733

Reply:
Whoops, that's annoying. We can make this part of the improvements post base-bump, then.

Reply:
https://github.com/stellar/js-stellar-base/issues/472